examples/language/bert/train_bert_demo.py

```diff
@@ -0,0 +1,332 @@
+import os
+from functools import partial
+from time import time
+
+import psutil
+import torch
+from packaging import version
+from torch import nn
+from torch.nn.parallel import DistributedDataParallel as DDP
+from transformers import AlbertConfig, AlbertForSequenceClassification, BertConfig, BertForSequenceClassification
+
+import colossalai
+from colossalai.logging import disable_existing_loggers, get_dist_logger
+from colossalai.nn.optimizer import HybridAdam
+from colossalai.nn.parallel import zero_model_wrapper, zero_optim_wrapper
+from colossalai.tensor import ColoParameter, ComputePattern, ComputeSpec, ProcessGroup, ReplicaSpec, ShardSpec
+from colossalai.utils import get_current_device
+from colossalai.utils.model.colo_init_context import ColoInitContext
+
+CAI_VERSION = colossalai.__version__
+
+
+def get_tflops(model_numel, batch_size, seq_len, step_time):
+    return model_numel * batch_size * seq_len * 8 / 1e12 / (step_time + 1e-12)
```

Comment:
I think we can move these utility functions into another file or move them to Colossalai if they are used in both GPT models and BERT models.

Reply:
Yes. But we want the user to run their examples in the current directory. Using common utils needs dependencies on its parent directory. You can fix them in another PR.

Reply:
We can just put these functions into Colossalai if necessary.

Reply:
OK, and just put them here is OK. Each example is an independent project.